docs/sources/alerting/fundamentals/alert-rules/alert-rule-types.md

```diff
@@ -10,24 +10,26 @@ weight: 102
 
 # Alert rule types
 
-Grafana supports several alert rule types, the following sections will explain their merits and demerits and help you choose the right alert type for your use case.
+Grafana supports several different alert rule types. Learn more about each of the alert rule types, how they work, and decide which one is best for your use case.
 
-## Grafana managed rules
+## Grafana-managed alert rules
 
-Grafana-managed rules are the most flexible alert rule type. They allow you to create alerts that can act on data from any of your existing data sources.
+Grafana-managed alert rules are the most flexible alert rule type. They allow you to create alerts that can act on data from any of our supported data sources.
 
-In addition to supporting any data source, you can add [expressions]({{< relref "/docs/grafana/latest/panels-visualizations/query-transform-data/expression-queries" >}}) to transform your data and express alert conditions.
+In addition to supporting multiple data sources, you can also add expressions to transform your data and set alert conditions. Using images in alert notifications is also supported. This is the only type of rule that allows alerting from multiple data sources in a single rule definition.
 
 ## Mimir, Loki and Cortex rules
 
-To create Mimir, Loki or Cortex alerts you must have a compatible Prometheus data source. You can check if your data source is compatible by testing the data source and checking the details if the ruler API is supported.
+To create Grafana Mimir or Grafana Loki-managed alert rules, you must have a compatible Prometheus or Loki data source.
 
-{{< figure src="/static/img/docs/alerting/unified/mimir-datasource-check.png" caption="Successfully connected to a Mimir Prometheus datasource" max-width="40%" >}}
+You can check if your data source supports rule creation via Grafana by testing the data source and observing if the ruler API is supported.
```

Comment:
I remember this being a confusing point for me. Is there somewhere we could link to for more info about the ruler API and how to check if it's supported?